apps/src/StudioApp.js

```diff
@@ -344,8 +344,8 @@ StudioApp.prototype.init = function (config) {
   ReactDOM.render(
     <Provider store={this.reduxStore}>
       <InstructionsDialogWrapper
-        showInstructionsDialog={(autoClose, showHints) => {
-            this.showInstructionsDialog_(config.level, autoClose, showHints);
+        showInstructionsDialog={(autoClose) => {
+            this.showInstructionsDialog_(config.level, autoClose);
```

Comment:
And [instructionsDialog.js](https://github.com/code-dot-org/code-dot-org/blob/84e8157350b422cc8fa3e0ab6772409999d8136e/apps/src/redux/instructionsDialog.js#L7)?